src/client.rs

```diff
@@ -85,10 +88,15 @@ impl SdkmsClientBuilder {
         let client = match self.client {
             Some(client) => client,
             None => {
-                let ssl = NativeTlsClient::new()?;
-                let connector = HttpsConnector::new(ssl);
-                let client = HyperClient::with_connector(Pool::with_connector(Default::default(), connector));
-                Arc::new(client)
+                #[cfg(feature = "native-tls")]
+                {
+                    let ssl = NativeTlsClient::new()?;
+                    let connector = HttpsConnector::new(ssl);
+                    let client = HyperClient::with_connector(Pool::with_connector(Default::default(), connector));
+                    Arc::new(client)
+                }
+                #[cfg(not(feature = "native-tls"))]
+                panic!("Unsuported");
```

Comment:
This should be more descriptive, e.g. `You should either provide a hyper Client or compile this crate with hyper-native-tls feature`

Reply:
Changed to the text you provided, thanks